raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -1536,15 +1533,27 @@ def test_mediator_must_not_send_lock_expired_when_channel_is_closed():
     assert search_for_item(iteration.events, SendLockExpired, {}) is None
 
 
-def test_mediator_lock_expired_with_receive_lock_expired():
+@pytest.fixture
```

Comment:
this could be a plain old function in my opinion

Reply:
True, I changed it.